dynamic-config/config-cc/src/main/java/org/apache/servicecomb/config/client/ConfigCenterClient.java

```diff
@@ -95,8 +95,8 @@
 
   private boolean isWatching = false;
 
-  private final ServiceLoader<AuthHeaderProvider> authHeaderProviders =
-      ServiceLoader.load(AuthHeaderProvider.class);
+  private final List<AuthHeaderProvider> authHeaderProviders =
+      SPIServiceUtils.getSortedService(AuthHeaderProvider.class);
```

Comment:
configure center is initialized before service center, will this cause problem?

Reply:
the dependency has some problem, we can refact later